packages/edit-post/src/components/layout/style.scss

```diff
@@ -7,14 +7,20 @@
 .edit-post-layout .components-editor-notices__snackbar {
 	position: fixed;
 	right: 0;
-	bottom: 40px;
-	padding-left: 16px;
-	padding-right: 16px;
+	bottom: $grid-unit-50;
+	padding-left: $grid-unit-20;
+	padding-right: $grid-unit-20;
 }
 
 .is-distraction-free {
 	.components-editor-notices__snackbar {
-		bottom: 20px;
+		bottom: calc(#{$grid-unit-50} / 2);
+	}
+}
+
+.has-secondary-sidebar {
+	.components-editor-notices__snackbar {
+		padding-left: calc(#{$grid-unit-20} + #{$nav-sidebar-width});
```

Comment:
Don't need to use `calc()` here, SCSS can add constants together at compile time.

```scss
padding-left: $grid-unit-20 + $nav-sidebar-width;
```